app/build.gradle

```diff
@@ -42,6 +42,10 @@ android {
     lintOptions {
         disable 'GoogleAppIndexingWarning'
     }
+
+    repositories {
+        mavenCentral()
```

Comment:
The repositories to use should be specified in the `build.gradle` in the root of the repo ([link](https://github.com/ericcornelissen/NervousFish/blob/develop/build.gradle#L15))